staging/app/ecs.tf

```diff
@@ -11,7 +11,8 @@ data "template_file" "musicbox-app" {
     fargate_cpu     = var.fargate_cpu
     fargate_memory  = var.fargate_memory
     aws_region      = var.aws_region
-    allowed_host    = aws_alb.staging.dns_name
+    command         = jsonencode(["passenger", "start", "-p", "80"])
+    allowed_hosts   = "^172\\\\.17\\\\.\\\\d{1,3}\\\\.\\\\d{1,3}$&^${aws_alb.staging.dns_name}$"
```

Comment:
This is the thing that allows the `172.17.x.x` ip range to talk to our web tasks.  We get the ip range based on what we've set for our staging VPC in `network.tf`.